### ISSUE TITLE
DataTable: Remove duplicate table headers (or change aria-labels) on scollable tables

### ISSUE THREAD

**Issue by Bukama** [primefaces/primefaces#9230]
### Description

[Primeface's DataTable Scroll](https://www.primefaces.org/showcase/ui/data/datatable/scroll.xhtml) render two tables with table headers. One of them gets the CSS class `ui-datatable-scrollable-theadclone` and the style `height: 0px;` which makes it invisble for regular users.

But screenreaders pick up the "hidden" table header and provide its content to the user, resulting in dublicate information and access options. On the screenshot you see the dublicate output of the `aria-label`s with the information to click for sorting ascending/descending.

The duplicated output results in accessibility problems and negative test results when testing agains the BITV 2.0 test / WCAG.

![primefaces_doubletable](https://user-images.githubusercontent.com/4354359/191889478-de7a5ae2-7764-42ad-b359-4048a586c02b.PNG)

### Describe the solution you would like

Simple solution: Remove the dublicate table header.
But I expect that the real solution is much complicated, because most probably there's a cause for the hidden table header. So maybe removing / changing the aria-labels of the hidden one could resolve the problem too.

### Additional context

Based on German and European laws/regulations webpages have to be accessible, following the Web Content Accessibility Guidelines (WCAG) of the W3. In short webpages have to fulfill the requirements of at least the second level ("AA") of the WCAG standard. Based on the EU regulations the German law defines that webpages of German government have to pass the "Barrierefreie-Informationstechnik-Verordnung - BITV 2.0" regulation, represented / verified by the ["BITV 2.0 test"](https://www.bitvtest.de/bitv_test.html).

So my team and I are working on making our web apps accessable, so they pass this test. The tests are executed by certified persons from another team. When analyising the results, we came to the conclusion that some violations can't be fixed by ourself only, but need changes in the Primeface's code. This issue is one of them. As my team and I couldn't find out, why some things are implemented the way they are, I created the issue as a feature request rather than a bug - even if, depending from the point of view, you could consider them as such.

Disclaimer: As guidelines say, I asked on the forum if such issues should be posted on Github and they were requested.

**Comment by melloware** [comment]
Yeah we need that cloned header for now but maybe stripping the aria values is a good idea.
